lib/nexmo_developer/app/services/orbit_feedback_notifier.rb

```diff
@@ -0,0 +1,49 @@
+class OrbitFeedbackNotifier
+  def self.call(feedback)
+    new(feedback).post!
+  end
+
+  def initialize(feedback)
+    @feedback = feedback
+  end
+
+  def params
+    @params ||= {
+      id: @feedback.id,
+      email: @feedback.owner.email,
+      resource: "Offered #{@feedback.sentiment} feedback on #{@feedback.resource.uri}",
+    }
+  end
+
+  def uri
+    @uri ||= URI("https://app.orbit.love/api/v1/#{ENV['ORBIT_WORKSPACE_ID']}/activities")
+  end
+
+  def post!
+    http = Net::HTTP.new(uri.host, uri.port)
```

Comment:
can we add a return clause here? in case the env variable is not defined, something like
``` 
return unless ENV['ORBIT_WORKSPACE_ID']
```

Reply:
Good catch!